webapp/src/components/post_type_jitsi/post_type_jitsi.tsx

```diff
@@ -0,0 +1,196 @@
+import * as React from 'react';
+
+import {Post} from 'mattermost-redux/types/posts';
+
+import Svgs from '../../constants/svgs';
+
+import {makeStyleFromTheme} from 'mattermost-redux/utils/theme_utils';
+
+export type Props = {
+    post?: Post,
+    theme: any,
```

Comment:
Is `any` the only reasonable way to do this?

Reply:
Actually, not, I didn't realize that there is a Theme type defined in mattermost redux, so I going to use it.